sdks/python/setup.py

```diff
@@ -129,6 +129,12 @@ def get_version():
     'typing>=3.6.0,<3.7.0; python_version < "3.5.0"',
     ]
 
+# [BEAM-8181] pyarrow cannot be installed on 32-bit Windows platforms.
```

Comment:
Is it possible to add this inline similar to the `'platform_system != "Windows"'` ?

Reply:
I looked into it and did not see a way to do that.

Reply:
This mirrors the condition in pyarrow: https://github.com/apache/arrow/blob/4f7ead867fdacbbe55ca4973f8e9979011df8011/python/setup.py#L271

Reply:
The closest approximation may be sys_platform == 'win32', but it will omit the second part of the condition, and I am not sure whether it is essential.

Reply:
See: https://www.python.org/dev/peps/pep-0508/#environment-markers